extensions/interactions/MathEquationInput/directives/math-equation-input-validation.service.ts

```diff
@@ -93,6 +95,12 @@ export class MathEquationInputValidationService {
 
         let splitInput = currentInput.split('=');
 
+        // Explicitly inserting '*' signs wherever necessary.
+        splitInput[0] = mathInteractionsService.insertMultiplicationSigns(
```

Comment:
Just to check -- do you already have a validation for "exactly one equals sign"?

Reply:
Hmm -- I've had this validation in the backend but for some reason in the frontend, I missed it. I've added it now along with tests, thanks for pointing this out!